docs/Location/Europe.md

```diff
@@ -11,3 +11,22 @@ Location : Germany
 Contact : something.host
 ```
 ---
+Hosted by @ [Thnks_CJ](https://github.com/ThnksCJ)
+```bash
+Host : lavalink.cjstevenson.xyz
+Port : 443
+Password : lookbehindyou
+Secure: True
+Location : UK
+Contact : Thnks_CJ#0001 or admin@cjstevenson.com
+```
```

Comment:
add another space it in, cause markdown = 2 space for thingy

Reply:
nvm it looks file 
![image](https://user-images.githubusercontent.com/55406094/139541466-5f355b9a-6ad1-4907-85cc-592be08f3762.png)